recipes/meds/meta.yaml

```diff
@@ -0,0 +1,46 @@
+{% set name = "meds" %}
+{% set version = "0.9.7" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/esheldon/meds/archive/v{{ version }}.tar.gz
+  sha256: 015fedb2754aec776c82528ab63a8157937400bcdb19ef9afb6838dac09ce431
+
+build:
+  number: 0
+  skip: True  # [win]
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+  host:
+    - python
+    - pip
+    - numpy
+  run:
+    - python
+    - fitsio
+    - pyyaml
+    - {{ pin_compatible('numpy') }}
+
+test:
+  imports:
+    - meds
+
+about:
+  home: https://github.com/esheldon/meds
+  license: GPL-3
+  license_family: GPL
+  license_file: LICENSE
+  summary: Python and C libraries to work with Multi Epoch Data Structures
+  doc_url: https://github.com/esheldon/meds
+  dev_url: https://github.com/esheldon/meds
+
+extra:
+  recipe-maintainers:
+    - beckermr
+    - esheldon
```

Comment:
@esheldon, would you like to help maintain this? We chatted offline but I did not ask.

Reply:
yep